userspace/libscap/scap.c

```diff
@@ -601,10 +601,10 @@ scap_t* scap_open_udig_int(char *error, int32_t *rc,
 	//
 	error[0] = '\0';
 	snprintf(filename, sizeof(filename), "%s/proc", scap_get_host_root());
-	if((*rc = scap_proc_scan_proc_dir(handle, filename, error)) != SCAP_SUCCESS)
+	if((*rc = scap_proc_scan_proc_dir(handle, filename, false, error)) != SCAP_SUCCESS)
```

Comment:
Why suppress=false here (and below, in scap_open_udig_int)? The process is exactly the same and tbh I'd just remove the parameter and always suppress scan errors (but see below)